packages/jest/src/create-enzyme-serializer.js

```diff
@@ -66,11 +66,13 @@ const wrappedEnzymeSerializer = {
 
 export function createEnzymeSerializer({
   classNameReplacer,
-  DOMElements = true
+  DOMElements = true,
+  includeStyles = true
```

Comment:
We might also need to add this in the `.d.ts` file

Reply:
`createEnzymeSerializer` uses the same `CreateSerializerOptions` interface as `createSerializer` (updated with `includeStyles` in #2709). In `enzyme.d.ts`, imported from `index.d.ts`. Or are you referring to a different `d.ts` file?

Reply:
No, this is fine. I was reviewing from mobile and I've assumed that there is another type for this one.